packages/blade/src/components/Accordion/_decisions/decisions.md

```diff
@@ -63,12 +127,30 @@ import { Accordion, AccordionItem } from '@razorpay/blade';
 
 ### AccordionItem
 
-| Key         | Type            | Default     | Description                                                                | Required |
-| ----------- | --------------- | ----------- | -------------------------------------------------------------------------- | -------- |
-| title       | `string`        | `undefined` | Title text content                                                         | ✅       |
-| description | `string`        | `undefined` | Body text content                                                          |          |
-| icon        | `IconComponent` | `undefined` | Renders a Blade icon as title prefix (requires `showNumberPrefix={false}`) |          |
-| children    | `JSX`           | `undefined` | Slot, renders any custom content                                           |          |
+| Key                         | Type              | Default     | Description                                                                                                                                     | Required |
+| --------------------------- | ----------------- | ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| title (⚠️ deprecated)       | `string`          | `undefined` | Title text content                                                                                                                              |          |
+| description (⚠️ deprecated) | `string`          | `undefined` | Body text content                                                                                                                               |          |
+| icon (⚠️ deprecated)        | `IconComponent`   | `undefined` | Renders a Blade icon as title prefix (requires `showNumberPrefix={false}`)                                                                      |          |
+| children                    | `React.ReactNode` | `undefined` | Accepts [`AccordionItemHeader`](#AccordionItemHeader) and [`AccordionItemBody`](#accordionitembody). Accepting other JSX elements is deprecated | ✅       |
+
+#### AccordionItemHeader
+
+| Key         | Type              | Default     | Description                                              | Required |
+| ----------- | ----------------- | ----------- | -------------------------------------------------------- | -------- |
+| title       | `string`          | `undefined` | Title text content                                       |          |
+| subtitle    | `string`          | `undefined` | Subtitle text content                                    |          |
+| titleSuffix | `React.ReactNode` | `undefined` | Suffix part of title. Mostly for Badge                   |          |
```

Comment:
how will we prevent people accidentally using titleSuffix and trailing interchangeable?

Reply:
It renders at 2 difference places. titleSuffix is immediately after title vs trailing is on the extreme right. So depending on the design they will use the correct parameter.

<img width="850" alt="image" src="https://github.com/razorpay/blade/assets/30949385/5c91827b-f4d7-49a7-8a08-3cfaee47438e">

<img width="807" alt="image" src="https://github.com/razorpay/blade/assets/30949385/1435cdf8-1f90-473c-9214-b76611c6d6d3">